lstm_regularization/model.py

```diff
@@ -0,0 +1,112 @@
+import torch
+import torch.nn as nn
+
+import torch.nn.functional as F
+from copy import deepcopy
+
+from lstm_regularization.weight_drop import WeightDrop
+from lstm_regularization.embed_regularize import embedded_dropout
+
+
+class LSTMBaseline(nn.Module):
+    def __init__(self, config):
+        super(LSTMBaseline, self).__init__()
+        dataset = config.dataset
+        target_class = config.target_class
+        self.is_bidirectional = config.bidirectional
+        self.has_bottleneck_layer = config.bottleneck_layer
+        self.mode = config.mode
+        self.TAR = config.TAR
+        self.beta_ema = config.beta_ema  ## Temporal averaging
+        self.wdrop = config.wdrop ## WEight dropping
+        self.embed_droprate = config.embed_droprate ## Embedding dropout 
+
+        input_channel = 1
+        if config.mode == 'rand':
+            rand_embed_init = torch.Tensor(config.words_num, config.words_dim).uniform_(-0.25, 0.25)
+            self.embed = nn.Embedding.from_pretrained(rand_embed_init, freeze=False)
+        elif config.mode == 'static':
+            self.static_embed = nn.Embedding.from_pretrained(dataset.TEXT_FIELD.vocab.vectors, freeze=True)
+        elif config.mode == 'non-static':
+            self.non_static_embed = nn.Embedding.from_pretrained(dataset.TEXT_FIELD.vocab.vectors, freeze=False)
+        else:
+            print("Unsupported Mode")
+            exit()
+
+        self.lstm = nn.LSTM(config.words_dim, config.hidden_dim, dropout=config.dropout, num_layers=config.num_layers,
+                            bidirectional=self.is_bidirectional, batch_first=True)
+        ################
```

Comment:
Don't need comments. Also PEP8 for `dropout = ...`

Reply:
Ported code from previous contributions for consistency. Nevertheless will change this. Thank you!
Will also remove the comments.

Reply:
Wrong line, see L41